site/en/getstarted/milvus_lite.md

```diff
@@ -43,9 +43,157 @@ from pymilvus import MilvusClient
+| [create_collection()](https://milvus.io/api-reference/pymilvus/v2.4.x/MilvusClient/Collections/create_collection.md)       | Support with limited parameters                               |
+| `collection_name`                                                                                                          | Y                                                             |
+| `dimension`                                                                                                                | Y                                                             |
+| `primary_field_name`                                                                                                       | Y                                                             |
+| `id_type`                                                                                                                  | Y                                                             |
+| `vector_field_name`                                                                                                        | Y                                                             |
+| `metric_type`                                                                                                              | Y                                                             |
+| `auto_id`                                                                                                                  | Y                                                             |
+| `schema`                                                                                                                   | Y                                                             |
+| `index_params`                                                                                                             | Y                                                             |
+| `enable_dynamic_field`                                                                                                     | Y                                                             |
+| `num_shards`                                                                                                               | N                                                             |
+| `partition_key_field`                                                                                                      | N                                                             |
+| `num_partitions`                                                                                                           | N                                                             |
+| `consistency_level`                                                                                                        | N (Only supports `Strong`; other levels are invalid)          |
```

Comment:
It should be that no matter what configuration, the effect is the same as Strong